### DISCUSSION TITLE
Is there a way to log weight?

### DISCUSSION THREAD

**Discussion by gramapp** [jlai/fitness-dashboard#21] [Q&A] [answered]
Is there a way to log weight?

**Comment by jlai** [comment]
There's now a log weight option from the + icon in the lower right of the dashboard

**Reply by gramapp** [reply]
Great, but small tiny issue for me its only available in kg and not lbs. Can lbs be available as units too?

**Reply by jlai** [reply, marked as answer]
What do you have set for the weight unit in https://dashboard.exercise.quest/settings ? If it's blank, you'll need to set it.

**Reply by gramapp** [reply]
Is there a link to the settings page in the dashboard?

**Reply by chrissard2** [reply]
yes, upper right hand corner, to the right of about, click on your photo

![image](https://github.com/user-attachments/assets/4b7cee23-e7a2-49b2-ae97-794a7249145b)

**Reply by gramapp** [reply]
@chrissard2 , thanks, did not see that.
